Source/Core/Core/PowerPC/JitInterface.cpp

```diff
@@ -40,6 +40,8 @@
 
 namespace JitInterface
 {
+JitBase* g_jit = nullptr;
```

Comment:
Why are both of these needed instead of just JitCommonBase, especially when JitCommonBase can reduce to a JitBase instance, since it derives from it.

Reply:
Because the PR queued up after this one will introduce a JIT that derives from `JitBase` but not `JitCommonBase`. That's what this whole thing is about. Though I guess I could downcast using dynamic_cast where necessary (unless that's disabled by compiler options or something).